devex/cross-nuget/standalone-builds/windows/build.ps1

```diff
@@ -0,0 +1,46 @@
+﻿# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Build for SGX
+# |- Default (with LVI)
+#    |- Debug
+#    |- Release
+
+Param(
+  [string]$SDK_PATH = 'SDK',
+  [string]$SDK_TAG = 'master'
+)
+
+$ErrorActionPreference = "Stop"
+
+If (-not (Test-Path -Path $SDK_PATH))
+{
+    git clone --recursive --depth=1 https://github.com/openenclave/openenclave SDK -b $OE_SDK_TAG
+    $SDK_PATH = (Join-Path -Path $PWD -ChildPath SDK)
+}
+Else
+{
+    $SDK_PATH = (Convert-Path $SDK_PATH)
+}
+
+If (Test-Path Build)
+{
+    Remove-Item Build -Recurse -Force
+}
+
+New-Item -ItemType Directory -Path \source\Build\Default\Debug | Out-Null
+New-Item -ItemType Directory -Path \source\Build\Default\Release | Out-Null
+
+Push-Location -Path \source\Build\Default\Debug
+cmake $SDK_PATH -G Ninja -DLVI_MITIGATION=ControlFlow -DNUGET_PACKAGE_PATH=C:\oe_prereqs -DBUILD_ENCLAVES=ON -DCPACK_GENERATOR=NuGet -DCMAKE_BUILD_TYPE=Debug
```

Comment:
I was under the impression that running cmake from a powershell script required calling in vcvars. I trust you have run and tested this and am approving but it might be worth a double check if you have an environment set up.

Reply:
Yes, vcvars is required, the script is called as such:
`vcvars64.bat && powershell -c "Set-ExecutionPolicy Bypass -Scope Process; .\build.ps1 -SDK_PATH \source\openenclave`

I'll add documentation to note this.